gslib/commands/config.py

```diff
@@ -1160,11 +1040,10 @@ def RunCommand(self):
       raise CommandException('Both -a and -e cannot be specified. Please see '
                              '"gsutil help config" for more information.')
 
-    if not configure_auth and (has_a or has_e or scopes or launch_browser):
+    if not configure_auth and (has_a or has_e):
```

Comment:
We are still parsing out the flags like -b, -f etc. Is it because we expect the error raised on line 796 to take care of them?

Reply:
LGTM, otherwise